src/aboutUs.json

```diff
@@ -0,0 +1,42 @@
+{
+    "sections": [
+      {
+        "title": "Introduction",
```

Comment:
the title should not be changed for any  i suppose.
 please detail the introduction 
 it will the content for about AnitaB.org

Reply:
Alright, I'll create more details for the introduction + content about AnitaB.org